Assignment2/david-jl-DavidJimenez.ttl

```diff
@@ -0,0 +1,19 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix schema: <http://schema.org/> .
+@prefix : <http://example.org/#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>. 
+
+:Class01 rdfs:includes :Sensor029;
+         rdfs:includes :Computer101.
+
+:Sensor029 rdfs:hasMeasurement :Measurement8401.
```

Comment:
But this is still wrong, and I have seen this problem earlier and reported about it.
You cannot use rdfs:XXX with a property that does not exist in RDFS